pyutilib/dev/runtests.py

```diff
@@ -131,6 +131,9 @@ def run(package, basedir, argv, use_exec=use_exec, env=None):
     else:
         cmd = ['nosetests']
 
+    if (platform == 'win' and sys.version_info[0:2] >= (3, 8)):
+        cmd.append('--traverse-namespace')
```

Comment:
Can you add some documentation for why this is needed?  A pointer to some python documentation that this was an intentional change for Python 3.8 on Windows (and indicates that it will not affect other versions / platforms)?

Reply:
Reading up on it more - I think it's actually a bug. Nose isn't maintained anymore (migrated to nose2), so we can't rely on them to fix the bug, but I found the answer to the problem on [a public forum](https://stackoverflow.com/questions/58556183/nose-unittest-discovery-broken-on-python-3-8). Would you like me to add a note in the code to that effect?

Reply:
Yes - we should definitely include a comment and the link.  (I can see myself completely forgetting this discussion and pulling my hair out when I come across this code again!)